### PR TITLE
Repackaged for maven

### PR DESCRIPTION
**Pull request by eugene-tds**

### CODE REVIEW

File: core/pom.xml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        <dependency>
+            <groupId>com.google.zxing</groupId>
+            <artifactId>core</artifactId>
+            <version>1.7</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.ant</groupId>
+            <artifactId>ant</artifactId>
+            <version>1.8.2</version>
+        </dependency>
```
Comment (lukaszlenart) [inline core/pom.xml:38]: Is there a direct dependency on the Ant API in the code?

**Review by lukaszlenart — COMMENTED** [review]
Looks good, just one doubt

File: core/pom.xml (hunk 1 above)
Comment (eugene-tds) [reply core/pom.xml:38]: Yeah, one of the classes (BarcodeTask) originally packaged into Barcode4J library requires it. The class can be used to generate barcodes from within Ant scripts. We can probably remove this class from the library and then dependency from Ant won't be needed, but as far as I can tell this class is present in released 2.1 version.

**Review by Grandys — CHANGES_REQUESTED** [review]
Github does not display this change, so I'll add comments here:
1. I'd extract common dependencies, from module-pom's to the parent pom in the dependencyManagement section. 
2. Does it make sense to extract `2.2.0` version into parent pom properties?

File: core/pom.xml (hunk 1 above)
Comment (lukaszlenart) [reply core/pom.xml:38]: It would be good to move this task into another module `barcode4j-ant`, but I think we can do it in another PR.

**Review by Grandys — APPROVED** [review]
LGTM 👍

**Pull request merged**